README.md

```diff
@@ -22,6 +22,12 @@ Netavark is a tool for configuring networking for Linux containers. Its features
 - [Podman](https://podman.io/docs) 4.0+
 - [protoc](https://grpc.io/docs/protoc-installation/)
 
+## MSRV (Minimum Supported Rust Version)
+
```

Comment:
Add a sentence here: "Netavark is guaranteed to build on this Rust version and later." or similar

Reply:
Given rust weird stance on backwards compatibility I find guaranteed to be a very strong word, i.e. it is not so recent that I think rust 1.80 failed to compile older version of the time crate
https://github.com/rust-lang/rust/issues/127343#issuecomment-2218261296

i.e. it is impossible to build aardvark-dns 1.10.0 with rust v1.80 or newer. To their credit they provide a helpful error but still
```
error[E0282]: type annotations needed for `Box<_>`
  --> /home/pholzing/.cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.30/src/format_description/parse/mod.rs:83:9
   |
83 |     let items = format_items
   |         ^^^^^
...
86 |     Ok(items.into())
   |              ---- type must be known at this point
   |
   = note: this is an inference error on crate `time` caused by an API change in Rust 1.80.0; update `time` to version `>=0.3.35` by calling `cargo update`
```

And given we to not test all rust versions there might be some weird stuff in between so I think I rather `s/guaranteed/supposed` in your sentence

Reply:
"We test that Netavark will build on the following Rust version. Newer versions should also build, and if they do not, the issue should be reported and will be fixed. Older versions are not guaranteed to build and issues will not be fixed"?

Reply:
sounds good